test/e2e/hooks_test.go

```diff
@@ -851,7 +851,7 @@ spec:
           template: http
     - name: http
       http:
-        url: http://dummy.restapiexample.com/api/v1/employees
+        url: https://github.com/
```

Comment:
Maybe we should call GitHub API endpoint?

Reply:
Non-blocking comment

Reply:
I felt that was likely to get rate limited, and we're not calling it with a token so it may randomly fail for that.

I considered just calling google instead.

Reply:
Sounds good

Reply:
I've remembered we've got our own http endpoint to test against already, `httpbin`, so switched to that.